src/Request.php

```diff
@@ -494,4 +499,20 @@ private function getFormatter()
     {
         return $this->client->getFormatter($this->getFormat());
     }
+
+    /**
+     * @return resource
+     */
+    public function getFile()
+    {
+        return $this->_file;
+    }
+
+    /**
+     * @param resource $file
+     */
+    public function setFile($file)
```

Comment:
It's easy to confuse it with a file meant for uploading it. I'd name it something like `setOutputFile()`.